doc/5_saving.md

```diff
@@ -97,6 +97,70 @@ val company = Company("DataStax", address)
 sc.parallelize(Seq(company)).saveToCassandra("test", "companies")
 ```
 
+## Specifying TTL and WRITETIME
+Spark Cassandra Connector saves the data without explicitly specifying TTL or WRITETIME. If a certain 
+values for them have to be used, there are a couple options provided by the API. 
+
+TTL and WRITETIME options are specified as properties of `WriteConf` object, which can be optionally 
+passed to `saveToCassandra` method. TTL and WRITETIME options are specified independently from one 
+another. 
+
+### Using a constant value for all rows
+When the same value should be used for all the rows, one can use the following syntax:
+
+```scala
+rdd.saveToCassandra(ks, tab, writeConf = WriteConf(ttl = TTLOption.constant(100)))
+rdd.saveToCassandra(ks, tab, writeConf = WriteConf(timestamp = TimestampOption.constant(ts)))
+```
+
+`TTLOption.constant` accepts one of the following values:
+  - `Int` / the number of seconds
+  - `scala.concurrent.duration.Duration`
+  - `org.joda.time.Duration`
+
+`TimestampOption.constant` accepts one of the following values:
+  - `Long` / the number of microseconds
+  - `java.util.Date`
+  - `org.joda.time.DateTime`
+
+### Using a different value for each row
+When a different value of TTL or WRITETIME has to be used for each row, one can use the following syntax:
+
+```scala
+rdd.saveToCassandra(ks, tab, writeConf = WriteConf(ttl = TTLOption.perRow("ttl")))
+rdd.saveToCassandra(ks, tab, writeConf = WriteConf(timestamp = TimestampOption.perRow("timestamp")))
+```
+
+`perRow(String)` method accepts a name of a property in each RDD item, which value will be used as TTL 
+or WRITETIME value for the row. 
+
+Say we have an RDD with `KeyValueWithTTL` objects, defined as follows:
+```scala
```

Comment:
Add "import com.datastax.spark.connector.writer._" so examples will work in Spark shell